cmd/server/assets/codes/bulk-issue.html

```diff
@@ -98,7 +99,7 @@
               <select class="form-control" id="sms-template">
                 <option value="Default SMS template">Default SMS template</option>
                 {{range $k, $v := $currentRealm.SMSTextAlternateTemplates}}
-                <option value="{{$k}}">{{$k}}</option>
+                <option value="{{$k}}" {{if eq $k $currentUser.DefaultSMSTemplateLabel}}selected{{end}}>{{$k}}</option>
```

Comment:
The `selectedIf` helper makes this easier (it's in render.go), but should just be `selectedIf eq ($currentUser.DefaultSMSTemplate $k)`